pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -998,6 +1058,58 @@ def test_clines_multiindex(clines, expected, env):
     assert expected in result
 
 
+@pytest.mark.parametrize(
+    "clines, expected",
+    [
+        (None, "1 \\\\\n2 \\\\\n3 \\\\\n4 \\\\\n"),
+        ("all;data", "1 \\\\\n2 \\\\\n3 \\\\\n4 \\\\\n"),
+        ("all;index", "1 \\\\\n2 \\\\\n3 \\\\\n4 \\\\\n"),
+        ("skip-last;data", "1 \\\\\n2 \\\\\n3 \\\\\n4 \\\\\n"),
+        ("skip-last;index", "1 \\\\\n2 \\\\\n3 \\\\\n4 \\\\\n"),
+        ("all-invisible;index", "1 \\\\\n2 \\\\\n3 \\\\\n4 \\\\\n"),
+        ("skip-last-invisible;index", "1 \\\\\n2 \\\\\n3 \\\\\n4 \\\\\n"),
+        (
+            "all-invisible;data",
+            dedent(
+                """\
+            1 \\\\
+            \\cline{1-1}
+            2 \\\\
+            \\cline{1-1} \\cline{1-1}
+            3 \\\\
+            \\cline{1-1}
+            4 \\\\
+            \\cline{1-1} \\cline{1-1}
+            """
+            ),
+        ),
+        (
+            "skip-last-invisible;data",
+            dedent(
+                """\
+            1 \\\\
+            2 \\\\
+            \\cline{1-1}
+            3 \\\\
+            4 \\\\
+            \\cline{1-1}
+            """
+            ),
+        ),
+    ],
+)
+@pytest.mark.parametrize("env", ["table"])
```

Comment:
Could you inline this? There's only 1 parameter

Reply:
I could—I was patterning this style based on the existing test that did this at line 1049; should that also change?

Reply:
its likely that other tests have copied and amended tests just like you have (sub optimally). Do worry about the others, just focus on making your changes as readable as possible.